File: Swift-C++-Testsuite/test/controlFlow/Swift/SwitchNumberTuples.swift
@main
struct MyMain {

    static func main() {
        let res = switchMe(input: (2, 3))
    }

    static func switchMe(input: (Int, Int)) -> Int {
        var counter = 0
        var max = 7
        switch input {
        case (1, 1):
            counter = max
        case (0, 1):
            counter = -max
        case (0, _):
            counter = -max
        default:
            counter = 42
        }
        return counter
    }
}

File: AutoTranspiledTests/try-2/SwitchNumberTuples.cpp

 struct MyMain 
{
	static int main()
	{
		int res = switchMe((2, 3));
		return res;
	}

	static int switchMe(pair<int, int>input)
	{
		int counter = 0;
		int max = 7;
		switch (input)
		{
			case make_pair(1, 1):
				counter = max;
				break; 
			case make_pair(0, 1):
				counter = -max;
				break; 
			case make_pair(0, _):
				counter = -max;
				break; 
			default: 
				counter = 42;
				break;		
		}
		return counter; 
	}
};